docs/project/refpack-assemblies.md

```diff
@@ -0,0 +1,71 @@
+# List of AspNetCore Refpack Assemblies
+
+- Microsoft.Extensions.Caching.Abstractions
+- Microsoft.Extensions.Caching.Memory
+- Microsoft.Extensions.Configuration
+- Microsoft.Extensions.Configuration.Abstractions
+- Microsoft.Extensions.Configuration.Binder
+- Microsoft.Extensions.Configuration.CommandLine
+- Microsoft.Extensions.Configuration.EnvironmentVariables
+- Microsoft.Extensions.Configuration.FileExtensions
+- Microsoft.Extensions.Configuration.Ini
+- Microsoft.Extensions.Configuration.Json
+- Microsoft.Extensions.Configuration.UserSecrets
+- Microsoft.Extensions.Configuration.Xml
+- Microsoft.Extensions.DependencyInjection
+- Microsoft.Extensions.DependencyInjection.Abstractions
+- Microsoft.Extensions.FileProviders.Abstractions
+- Microsoft.Extensions.FileProviders.Composite
+- Microsoft.Extensions.FileProviders.Embedded
+- Microsoft.Extensions.FileProviders.Physical
+- Microsoft.Extensions.FileSystemGlobbing
+- Microsoft.Extensions.Hosting
+- Microsoft.Extensions.Hosting.Abstractions
+- Microsoft.Extensions.Http
+- Microsoft.Extensions.Logging
+- Microsoft.Extensions.Logging.Abstractions
+- Microsoft.Extensions.Logging.Configuration
+- Microsoft.Extensions.Logging.Console
+- Microsoft.Extensions.Logging.Debug
+- Microsoft.Extensions.Logging.EventLog
+- Microsoft.Extensions.Logging.EventSource
+- Microsoft.Extensions.Logging.TraceSource
+- Microsoft.Extensions.Options
+- Microsoft.Extensions.Options.ConfigurationExtensions
+- Microsoft.Extensions.Options.DataAnnotations
+- Microsoft.Extensions.Primitives
+- Microsoft.Extensions.WebEncoders
+- Microsoft.Win32.Registry
+- System.Diagnostics.EventLog
+- System.IO.Pipelines
+- System.Security.AccessControl
+- System.Security.Cryptography.Cng
+- System.Security.Cryptography.Xml
+- System.Security.Permissions
+- System.Security.Principal.Windows
+- System.Windows.Extensions
+
+# List of WindowsDesktop Refpack Assemblies
+
+- Microsoft.Win32.Registry
```

Comment:
@anipik are we missing System.Speech?

Reply:
i dont think we shipped a new version of the system.speech ref pack in 5.0 servicing releases, but i will verify.

Reply:
@danmoseley this is referring winforms/wpf ref pack not to the Windows Compatibility pack, is System.Speech in the wpf/winforms ref pack?

Reply:
Oh - not if I had to do something special to make it happen.
But when I take a regular .NET Core project and reference the compat pack, I get intellisense for speech. So I assumed some magic was happening.